nixos/modules/system/boot/systemd.nix

```diff
@@ -725,6 +725,8 @@ in
 
       "systemd/system.conf".text = ''
         [Manager]
+        DefaultCPUAccounting=yes
+        DefaultBlockIOAccounting=yes
```

Comment:
All the `BlockIO*` directives are now just `IO*`, so you can use `DefaultIOAccounting`.